src/main/java/com/gregori/order_item/domain/OrderItem.java

```diff
@@ -0,0 +1,54 @@
+package com.gregori.order_item.domain;
+
+import com.gregori.common.AbstractEntity;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class OrderItem extends AbstractEntity {
+	private Long id;
+	private Long orderId;
+	private Long orderCount;
+	private Long itemId;
+	private String itemName;
+	private Long itemPrice;
+	private Status status;
+
+	@Getter
+	@RequiredArgsConstructor
+	public enum Status {
+		PAYMENT_COMPLETED("결제 완료"), SHIPMENT_PREPARATION("배송 준비중"),
+		SHIPPED("배송중"), DELIVERED("배송 완료");
```

Comment:
```suggestion
		PAYMENT_COMPLETED("결제 완료"), 
		SHIPMENT_PREPARATION("배송 준비중"),
		SHIPPED("배송중"), 
		DELIVERED("배송 완료");
```

정말 별거 아닙니다 ㅎ 이렇게 하면 쬐금 더 가독성이 좋아지지 않을까 하여...